spec/restaurants_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'dishes'
+require 'restaurant'
+
+describe 'restaurant' do
+
+	subject { Restaurant.new }
+
+	it 'creates a restaurant' do
+		expect(subject).to be_kind_of Restaurant
+	end 
+
+	it 'creates a menu' do
+		expect(subject.menu).to 
+	end
+
+	it 'create a '
```

Comment:
Det här är helt enkelt slarvigt Daniel. Om du inte lägger till ett `do .. end` så måste du åtminstone komma ihåg att markera ett ofärdigt test med `xit` istället för `it`